packages/core/src/Diagram.ts

```diff
@@ -83,6 +83,38 @@ export class Diagram {
     return this.linksAtOutputPortId(port.id)
   }
 
+  getNodeIdAndPortIdFromLinkId(linkId: string): { nodeId: string, portId: string } {
```

Comment:
Is this method ever used?

Reply:
This method is currently not in use. It was created to retrieve information like items using the nodeId and portId format. If the `Observability improvements` task is completed and this method is still unnecessary, I will remove it ❀